resources/run_adf.sh

```diff
@@ -53,11 +56,12 @@ LOGS_PATH=$LOCAL_INSTALL_DIR/logs
 DB_CONFIG_PATH=$(pwd)/resources/integration_test/testdata/adf_db_config.json
 MONGOD_PORT=28017
 MONGOHOUSED_PORT=27017
+export COMPUTE_MODE_MONGODB_PORT=47017
 START="start"
 STOP="stop"
 MONGOD="mongod"
 MONGOHOUSED="mongohoused"
-TENANT_CONFIG="./testdata/config/mongodb_local/tenant-config.json"
+TENANT_CONFIG="./testdata/config/inline_local/tenant-config.json"
```

Comment:
I see they still have mongodb_local/inline_local, was poking around the datasets but wondering what precipitated this change?

Reply:
I updated this based off of their README. Locally, the `inline_local` always works to start it up, but having issues with `mongodb_local`.